exonum-java-binding/cryptocurrency-demo/start-cryptocurrency-demo.sh

```diff
@@ -13,43 +13,37 @@ set -eu -o pipefail
 
 header "DETECTING ENVIRONMENT"
 
-# EXONUM_HOME shoudl be set to the root directory of Exonum Java App installed
-if [ -z "${EXONUM_HOME+x}" ];
+# Check whether the Exonum Java App is installed.
+EXONUM_JAVA_APP="exonum-java"
+command -v ${EXONUM_JAVA_APP} >/dev/null 2>&1 || { echo >&2 "Please install the Exonum Java App and make sure that 'exonum-java' binary is available via PATH. Aborting."; exit 1; }
+
+# Check that path to the JVM library is set.
+if [ -z "${JAVA_LIB_DIR+x}" ];
 then
-    echo "ERROR: EXONUM_HOME variable is empty. Please point the EXONUM_HOME to the root of installed Exonum Java App."
+    echo "ERROR: JAVA_LIB_DIR variable is not set. Please make sure you executed 'source ../tests_profile'"
```

Comment:
Please see the task description: https://jira.bf.local/browse/ECR-3093

It must work with an installed application, not with a script for our **internal** test purposes that, not accidentally, requires *Rust* installed.

Reply:
I'd say such detailed information should be initially presented in the task to avoid ambiguity.